pulpcore/cli/core/orphan.py

```diff
@@ -47,7 +47,7 @@ def orphan(ctx: click.Context, pulp_ctx: PulpCLIContext) -> None:
     ),
     needs_plugins=[PluginRequirement("core", specifier=">=3.15.0")],
 )
-@pass_entity_context
+@pass_view_set_context
 @pass_pulp_context
 def cleanup(pulp_ctx: PulpCLIContext, orphan_ctx: PulpEntityContext, **kwargs: t.Any) -> None:
```

Comment:
```suggestion
def cleanup(pulp_ctx: PulpCLIContext, orphan_ctx: PulpViewSetContext, **kwargs: t.Any) -> None:
```

Am I right?

Reply:
Indeed. In fact, if this still works something else must be wrong!?